Automattic/wp-calypso
# Add the Pressable promotion banner under the plans grid

## Proposed Changes

This PR implements the Pressable promotional banner under the pricing grid. For more context, please refer to p9Jlb4-9Dn-p2#comment-9916.

In general,
* Figma: Uzd0AYiDAV4eFq1V35T9Ef-fi . See the Multi-site section.
* The banner is only shown in `/start/plans` and only for the English locale since the translation isn't completed.
* Add two tracking events: `calypso_multisite_promo_banner_impression` and `calypso_multisite_promo_banner_cta_click`. Since there is a plan to show banners of other partners, they are defined as general events with a `service` prop. 
* Clicking on the CTA will lead to the corresponding Pressable landing page with UTM arguments suggested in https://github.com/Automattic/wp-calypso/pull/84636#issuecomment-1832746596 and the follow-up: https://github.com/Automattic/wp-calypso/pull/84636#issuecomment-1833222608. 
* The right copy of the banner can also be found in https://github.com/Automattic/wp-calypso/pull/84636#issuecomment-1832746596
* The banner is loaded asynchronously so it won't impact the bundle size of where it doesn't show.

### Media

Desktop:
<img width="1253" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1842898/5a88c1e9-fec6-426a-aae6-eba28ed3e002">

Mobile:
<img width="439" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1842898/db3178ac-7841-47ed-b92b-130bf6b36d21">


## Testing Instructions



* Visit /start/plans in a English locale, the banner should show.
* Test out different resolutions, and make sure the banner is responsive according to the design.
* Expand/collapse the feature comparison table. The banner should stay at the bottom accordingly.
* Visit other flows, e.g. `/setup/newsletter`, `/start/launch-site`, `/start/domain`. The banner shouldn't show.
* Visit /start/plans in a non-English locale, the banner shouldn't show.

## Pre-merge Checklist
